src/test/fixtures/results/sample_order.js

```diff
@@ -1,4 +1,4 @@
-export default {
+const defaultResponse = {
```

Comment:
You could also keep this as is and add fulfillments to the js object in the test where it is needed, instead of having the below function, if you thing it is simpler.

Reply:
i thought about it but then we had to merge the `fulfillments` somewhere in nested hash, right?